react-app-rewired/headers/csps/defi/swappers/Lifi.ts

```diff
@@ -4,5 +4,7 @@ export const csp: Csp = {
   'connect-src': [
     // src/lib/swapper/LifiSwapper/LifiSwapper.ts
     'https://li.quest',
+    'https://mainnet.infura.io',
```

Comment:
Will this be borked on localhost? FOXy is currently borked locally precisely because of that - requests to Infura are only allowed from `*.shapeshift.com` domains.
Not sure what fires these, but gut feel is if we broadcast these on our own node, we shouldn't need to add a connect-src directive for Infura

Reply:
Lifi sdk sends requests to infura, and it seems that its required to execute trades. It's required to work from localhost (and does work - without this local is broken), not sure about our shapeshift domains though

Reply:
@gomesalexandre it's possible the LiFi SDK has a proxy wrapper around the requests to handle `Access-Control-Allow-Origin`.